CHANGELOG.md

```diff
@@ -25,3 +31,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix bug with standard library includes.
 ### Removed
 
+[Unreleased]: https://github.com/aeternity/aesophia_cli/compare/v4.3.0...HEAD
+[4.2.1]: https://github.com/aeternity/aesophia_cli/compare/v4.2.0...v4.3.0
```

Comment:
still the range is wrong

Reply:
aaaaaaaaaaa

Reply:
No the version is wrong, there is no 4.2.1 for the cli?

Reply:
Should be good now, checked the links